src/_scss/pages/homepage/hero/hero.scss

```diff
@@ -50,7 +50,7 @@
         }
 
         strong.homepage-hero__headline_weight_bold {
```

Comment:
Should we rename this class now?

Reply:
I don't think so because it is still relatively bold